corehq/toggles.py

```diff
@@ -1597,3 +1597,11 @@ def _commtrackify(domain_name, toggle_is_enabled):
     notification_emails=['jemord'],
     help_link='https://confluence.dimagi.com/display/RD/Related+Locations',
 )
+
+ICDS_DISHA_API = StaticToggle(
+    'icds_disha_access',
+    'ICDS: Access DISHA API',
+    TAG_CUSTOM,
+    namespaces=[NAMESPACE_USER],
+    relevant_environments={'icds', 'icds-new', 'softlayer'},
```

Comment:
`icds` doesn't exist anymore. can use `icds-new`. this will also need to include the new environment

Reply:
Sure. Will add the new one when it comes online